distributed-databases/cloudnative-postgres/examples/storageclass.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: ebs-sc
+provisioner: ebs.csi.aws.com
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+parameters:
+  csi.storage.k8s.io/fstype: xfs
+  type: gp3
```

Comment:
Lets use Provisioned IOPS SSD (io2) Volumes
They are useful for cases which are latency-sensitive, like large database workloads (e.g., MySQL, Cassandra, MongoDB, and Oracle) and critical business applications that need the kind of sustained performance gp3 volumes can’t achieve.